### DISCUSSION TITLE
Screengrab. Add current date to output directory

### DISCUSSION THREAD

**Discussion by anna-poddubnaya-gismart** [fastlane/fastlane#17643] [Q&A] [answered]
Hi!
I'm using screengrad for ui screenshot test.
I run command `fastlane screengrab --output_directory '/Users/User/fastlane'` to save all output in current directory. 
I would like to save test result in directory with current date, i.e. '/Users/User/fastlane/20-11-2020'
I've already tried `fastlane screengrab --output_directory '/Users/User/fastlane/$(date +%Y%m%d)'` but it doesn't work.
**How can I add date to the end of the path using some pattern?** I don't want to print it every time manually.
Thanks!

**Comment by rogerluan** [comment, marked as answer]
Hey @AnyaPoddubnaya 👋 

Short answer: there's no way. You'll need a new option to be implemented in `screengrab` to allow that to happen. 

If you feel like implementing this, feel free to open a PR 😊
